scripts/Muon/GUI/Common/plot_widget/base_pane/base_pane_model.py

```diff
@@ -0,0 +1,46 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
```

Comment:
This is very picky but we can change the time stamp in these files to 2021 if we want